code/nss-patches/patch.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+cwd=$(cd $(dirname $0); pwd -P)
+
+# Add license headers to all files.
+for file in `find "$cwd/../../snapshots/nss/" -type f`; do
+    echo -e "$(cat $cwd/license.txt)\n\n$(cat $file)" > "$file"
+done
+
+# Patch files where file names changed.
+patch "$cwd/../../snapshots/nss/hacl_curve25519_64.h" < "$cwd/hacl_curve25519_64.h.patch"
```

Comment:
I tried your branch and it got rejected on my machine...
```
patch: **** rejecting file name with ".." component: /opt/hacl-star/code/nss-patches/../../snapshots/nss/hacl_curve25519_64.h
patch: **** rejecting file name with ".." component: /opt/hacl-star/code/nss-patches/../../snapshots/nss/hacl_curve25519_64.c
```

Reply:
Hm, weird. How did you call the make file?

Reply:
is it a gnu vs bsd utility issue?

Reply:
Hm, I can't really test this right now because `make prepare` isn't working on Mac.